dbt_subprojects/daily_spellbook/models/relay/schema.yml

```diff
@@ -0,0 +1,55 @@
+version: 2
+
+sources:
+  - name: evms
+    description: "EVM transaction and token transfer data"
+    tables:
+      - name: erc20_transfers
+        description: "ERC-20 token transfer events across chains"
```

Comment:
we shouldn't have sources in this file. just a repo-wide design principle.

you can add `evms` objects [here](https://github.com/duneanalytics/spellbook/blob/main/sources/_subprojects_outputs/daily_spellbook/_sources.yml#L26-L39) instead